README.md

```diff
@@ -1,45 +1,76 @@
 # Sentry Release GitHub Action
-Automatically create a Sentry Release in a workflow.
-
-## Why Releases?
-A release is a version of your code that is deployed to an environment. 
-When you give Sentry information about your releases, you unlock a number of new features:
- - Determine the issues and regressions introduced in a new release
- - Predict which commit caused an issue and who is likely responsible
- - Resolve issues by including the issue number in your commit message
- - Receive email notifications when your code gets deployed
-
-Additionally, releases are used for applying source maps to minified JavaScript to view original, untransformed source code.
+Automatically create a Sentry release in a workflow. 
+You can learn more about releases in [releases documentation](https://docs.sentry.io/workflow/releases).
 
 ## Usage
 Adding the following to your workflow will create a new Sentry release for the 
-`production` environment of your project and upload JavaScript source maps from
-the `./lib` directory. 
+`production` environment of your project with an automatically generated `version` name.
   
 ```yaml
-uses: getsentry/action-release@v1
-with:
-  environment: 'production'
-  sourcemaps: './lib'
+jobs:
+  release:
+    runs-on: ubuntu-latest
+    env:
+      SENTRY_AUTH_TOKEN: ${{ secrets.SENTRY_AUTH_TOKEN }}
+      SENTRY_ORG: ${{ secrets.SENTRY_ORG }}
+      SENTRY_PROJECT: ${{ secrets.SENTRY_PROJECT }}
+    steps:
+    - uses: actions/checkout@v2
+    - uses: getsentry/action-release@v1.0.0
```

Comment:
Can can we add the name parameter for our action?

Also do we need to define the environment in the top level? Can we bundle this under the Sentry step?

Reply:
👍